src/Our.Umbraco.Ditto/ComponentModel/Attributes/DittoLazyAttribute.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Our.Umbraco.Ditto.ComponentModel.Attributes
+{
+    /// <summary>
+    /// The Ditto lazy property attribute.
+    /// Used for specifying that Ditto should lazy load this property.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Class, AllowMultiple = false)]
```

Comment:
With this approach, would we still want a Ditto.RegisterLazyAttribute function to register the class level ditto lazy on types you may not have control over? (again, this would just make that types inner virtual properties lazy, not the type itself)